### DISCUSSION TITLE
Primary channel - not included in Settings

### DISCUSSION THREAD

**Discussion by ErikS-web** [egzumer/uvk5-chirp-driver#7] [Ideas]
When loading a channel list, the Primary Channel is also sent to the radio.
I discovered this because I didn't have it the same everywhere in saved channel lists.

But I don't see a Settings to display/change this setting?

Can be easily tested.
Load a list into the radio.
Change primary channel with A/B and see where the triangle is.
Reload the list into the radio and see where the triangle is.

**Comment by ErikS-web** [comment]
Also missing.... Initial VFO-Frequency/Memory-channel for VFO- A and B

**Reply by joc2** [reply]
ok, i see it, it will be add in basic display setting soon
